playbooks/roles/configure/tasks/main.yml

```diff
@@ -117,15 +119,38 @@
 - import_role:
     name: configure
     tasks_from: configure_client_certificates
+  vars:
+    certificate: "{{ work_dir_remote }}/USER-cert.cer"
+    dataset: USER
+    label: API ML Client
 
-
+- import_role:
+    name: configure
+    tasks_from: configure_client_certificates
+  vars:
+    certificate: "{{ zowe_keystore_dir }}/{{ zowe_keystore_alias }}/{{ zowe_keystore_alias }}.keystore.jwtsecret.pem"
+    dataset: JWTSCRT
+    label: jwtsecret
+  when: zos_keystore_mode is undefined or zos_keystore_mode != 'KEYSTORE_MODE_KEYRING'
 # ============================================================================
 # Setup certificates for keyring (self-signed)
 - import_role:
     name: configure
     tasks_from: setup_keyring_certificates
   when: zos_keystore_mode is defined and zos_keystore_mode == 'KEYSTORE_MODE_KEYRING'
 
+- name: Create zowe token
+  raw: tsocmd "RACDCERT ADDTOKEN({{ zowe_token_name }})"
+  when: zos_security_system == 'RACF'
+
+- name: Bind jwtsecret to profile
+  raw: tsocmd "RACDCERT BIND(TOKEN({{ zowe_token_name }}) ID({{ zowe_runtime_user }}) LABEL('{{ zowe_token_label }}'))"
+  when: zos_security_system == 'RACF'
+
+- name: List zowe token certificates
+  raw: tsocmd "RACDCERT LISTTOKEN({{ zowe_token_name }})"
+  when: zos_security_system == 'RACF'
```

Comment:
First, we can put these into a block like this:

```yaml
- name: Configure Zowe JWT token
  when: zos_security_system == 'RACF'
  block:
  - name: Create zowe token
    raw: tsocmd "RACDCERT ADDTOKEN({{ zowe_token_name }})"
  - name: Bind jwtsecret to profile
    raw: tsocmd "RACDCERT BIND(TOKEN({{ zowe_token_name }}) ID({{ zowe_runtime_user }}) LABEL('{{ zowe_token_label }}'))"
  - name: List zowe token certificates
    raw: tsocmd "RACDCERT LISTTOKEN({{ zowe_token_name }})"
```

Second, the check `zos_security_system == 'RACF'` may allow this section to run on other RACF systems other than Marist-1. This may cause failures because `zowe_token_name` and `zowe_token_label` are empty. Maybe checking `zowe_apiml_security_x509_enabled` is better.

Third, I saw this `main.yml` has many steps related to certificate, keyring. So would suggest to put related tasks into a separated file and then import from `main.yml`.